bitnami/cilium/templates/configmap.yaml

```diff
@@ -34,9 +34,10 @@ enable-ipv4: "true"
 enable-ipv6: "false"
 # IP Address Management (IPAM)
 # https://docs.cilium.io/en/stable/network/concepts/ipam
-{{- if .Values.agent.enableENI }}
+{{- if or .Values.aws.enabled (and .Values.azure.enabled (not .Values.agent.byoCNI)) }}
 routing-mode: "native"
 enable-endpoint-routes: "true"
+ipv4-native-routing-cidr: "10.0.0.0/8"
```

Comment:
Wouldn't adding a new value instead of hardcoding this make sense?

From upstream docs: https://github.com/cilium/cilium/blob/e13fa8b4bab6e10e521060ae6a8cfb6e8c4d0fea/Documentation/network/clustermesh/clustermesh.rst#additional-requirements-for-native-routed-datapath-modes

Reply:
Overwriting that parameter (or any other) would be as easy as setting:
```yaml
overrideConfiguration:
  ipv4-native-routing-cidr: "10.0.0.0/16"
```
We've created this mechanism to avoid exposing parameters for the big amount of supported configuration options.

Reply:
In that case, looks good to me!